03.asynchronous/callback/error.js

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env node
+
+import sqlite3 from "sqlite3";
+
+const db = new sqlite3.Database(":memory:");
+
+db.run(
+  `CREATE TABLE books (id INTEGER PRIMARY KEY AUTOINCREMENT, title TEXT NOT NULL UNIQUE)`,
+  () => {
+    db.run("INSERT INTO books (title) VALUES (null)", (err) => {
```

Comment:
指摘が漏れていましたが、`null` は SQL のキーワードなので大文字にしてください。

```suggestion
    db.run("INSERT INTO books (title) VALUES (NULL)", (err) => {
```

Reply:
大文字にしました